### PR TITLE
[new] System requirements for NodeJS API

### PR DESCRIPTION
**Pull request by Annyv2**
Added system requirements for NodeJS API tutorial

### CODE REVIEW

File: articles/server-apis/nodejs.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ snippets:
 
 ## NodeJS API Tutorial
 
+You can get started by either downloading the seed project or if you would like to add Auth0 to an existing application you can follow the tutorial steps.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* NodeJS 4.2
```
Comment (ntotten) [inline articles/server-apis/nodejs.md:27]: Do we use express or anything else? We need to mention versions for major libraries as well.

Comment (Annyv2) [inline articles/server-apis/nodejs.md:27]: I didn't know that, I'll fix it

Comment (ntotten) [inline articles/server-apis/nodejs.md:27]: @Annyv2 Ping :)

Comment (Annyv2) [inline articles/server-apis/nodejs.md:27]: @ntotten Hi! I added express already, can you see it? do you want me to add anything else?

**Pull request merged**